legal-api/src/legal_api/reports/business_document.py

```diff
@@ -568,7 +568,10 @@ def _get_summary_display_name(filing_type: str,
                                   filing_sub_type: Optional[str],
                                   legal_type: Optional[str]) -> str:
         if filing_type == 'dissolution':
-            return BusinessDocument.FILING_SUMMARY_DISPLAY_NAME[filing_type][filing_sub_type][legal_type]
+            if filing_sub_type == 'voluntary' and legal_type in ['SP', 'GP']:
+                return 'Dissolution Application'
+            else:
+                return BusinessDocument.FILING_SUMMARY_DISPLAY_NAME[filing_type][filing_sub_type]
```

Comment:
This will work but it not extensible. Can you try a fallback method.
Ref:
https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/reports/report.py#L236
https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/reports/report.py#L1420

Reply:
Okay. Will do

Reply:
Will refactor to an extensible code when we work on this later